openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion/__init__.py

```diff
@@ -2,8 +2,10 @@
 """
 Discussion XBlock
 """
+from __future__ import absolute_import
 import logging
-import urllib
+import six
+from six.moves import urllib
```

Comment:
Similar suggestion here as with `urlparse` above, probably better to just disable the pylint warning than to do the extra namespace lookups in the function body.